### ISSUE TITLE
Delay continues to grow in measureCOSensor

### ISSUE THREAD

**Issue by ellispritchard** [jenfoxbot/HazardousGasMonitor#1]
`startMillis` is set in `setup()`, and never gets reset, so the delay simply grows forever:
https://github.com/jenfoxbot/HazardousGasMonitor/blob/04e5f5337f219ce195c079c45969bb7443979dc8/HazGasMonitor#L215-L216

Also, the comment `(just before switching to 1.5V)` doesn't seem to apply, since the CO sensor is always read every loop, regardless of heater state. Should it only be read before switching back to low phase?

**Comment by chupocro** [comment]
Yes, you are right, that funcion for measuring CO doesn't have any sense and the program seems uncompleted. It's even using pin D6 which isn't connected.

I've been using MQ-7 for years, you may check the description of how it has to be used I wrote a few hours ago here in the comments:

https://www.instructables.com/Arduino-CO-Monitor-Using-MQ-7-Sensor/
